sdk-lib/src/main/java/cash/z/ecc/android/sdk/internal/db/derived/DerivedDataDb.kt

```diff
@@ -77,7 +77,11 @@ internal class DerivedDataDb private constructor(
                 }
                 repeat(missingAccounts) {
                     runCatching {
-                        backend.createAccountAndGetSpendingKey(checkedSeed, checkpoint, recoverUntil)
+                        backend.createAccountAndGetSpendingKey(
+                            seed = checkedSeed,
+                            treeState = checkpoint.treeState().encoded,
```

Comment:
This should probably be `checkpoint.treeState()` instead, since this is intended to be the typesafe API; sorry, I forgot to trim the `.encoded` when I copy-pasted the line.

Reply:
Good idea, the `createAccountAndGetSpendingKey` API updated.